Makefile

```diff
@@ -63,6 +63,12 @@ ifeq ($(processor),$(filter $(processor),aarch64 arm64 arm armv7l))
 endif
 	$(EXEC_WRAPPER) $^
 
+check-crypto: tests/main
```

Comment:
How about accepting a variable `FEATURES` which can be recognized as the corresponding flags?
See https://github.com/sysprog21/rv32emu/blob/master/Makefile#L69

Sample usage:
```shell
make FEATURE=crypto clean all
make FEATURE=crypto,crc clean all
```

Reply:
it seems we need to exclude `armv7` from testing crypto flag?

Reply:
ARMv8 introduced hardware acceleration for cryptography. From the perspective of Armv8, it is possible to use the cores with 32-bit mode only. That is, A32, which allows the use of cryptographic extension as well.

To make it clear, we should consider the following targets:
* Armv7-A: 32-bit. NEON is the necessary feature for SSE2NEON.
* Armv8-A: 64-bit and/or 32-bit. It comes with several extensions such as cryptography and CRC.

See also: https://github.com/ThomasKaiser/sbc-bench/blob/master/results/ARMv8-Crypto-Extensions.md